### PR TITLE
Wall E pull request #2

### PR DESCRIPTION
**Pull request by Divesh2**

### CODE REVIEW

**Comment by agshubhs** [comment]
@Divesh2  Following changes are needed:-
1. Put Android App folder inside **src** folder.
2. Write all the subsection in readme file using numbered list rather than paragraph.
3. Add Images of Solidwork designs in Mechanical Design Folder.
4. Resize the image.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,67 @@
 # Wall-E
 Srishti 2019
+## IMAGE
```
Comment (agshubhs) [inline README.md]: Don't put the title IMAGE

File: README.md [hunk 2]
```diff
@@ -1,2 +1,67 @@
...
+With the help of these wheels, the bot can move in all directions.The hands were operated using three dc motors.The "up-down" motion of hands were provided  using worm and spur gear mechanism.The hands were able to converge or diverge with the help of pulley and belt mechanism.
+The body was made using channels and plywood whereas the head was 3D printed.
+
+
+## ELECTRONICS ASPECT OF THE DESIGN
+We used two microcontrollers namely Arduino Uno and Arduino Nano, two L298n motor drivers, cytron motor driver,dc motors, servo motor,  bluetooth module and two LED Matrix MAX7219 for the this bot.Uno was used for movements of wheels and hands whereas nano was used for operating the head including LED Matrix used as the eyes of our bot, WallE.
+The arduino code can be found in "src" folder. Our bot was controlled through an Android device using an app made on MITAppInventor2. A bluetooth module was used for transmission of commands from android device to our bot. All the dc motors were powered through Li-Po dc battery.
+
+
+## COST STRUCTURE
```
Comment (agshubhs) [inline README.md]: Make a Detailed Table for cost structure for price of major parts.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,67 @@
...
+Currently, our bot is manually operated and thus require human help to be operated.
+
+
+## FUTURE IMPROVEMENTS
+Many furthur improvements can be done in this robot. Using IOT, this bot can be controlled from anywhere and can be live-streamed using webcam.
+It can be made autonomous using ML so that it can detect dustbins and collect garbage.Hence, it will no longer require commands to move, pick or dump.
+
+
+
+## TEAM MEMBERS
```
Comment (agshubhs) [inline README.md]: Write this as a numbered list like 
1.   
2.   
3.

**Comment by Divesh2** [comment]
@Shubhanshu07 
Sir, what should be the size of images?

**Comment by agshubhs** [comment]
@Divesh2 
1. The first image should be half of its present size.
2. Make a table for the Cost structure.
3. Write all the headings normally not in capital letters.
4. Make a separate folder of android App inside src folder
5. Commit mesage shoulde be clear not like delete q , delete r, create s

File: README.md [hunk 4]
```diff
@@ -1,2 +1,79 @@
...
+|L298n motor driver(2)|320|
+|Cytron motor driver|2000|
+|Bluetooth module|350|
+|Voltmeter|160|
+|Servo motor(s3003)|400|
+|DC motors(5)|3000|
+|Li-Po battery(2)|2200|
+|Led Matrix MAX7219(2)|300|
+|Other Components|2000|
+
```
Comment (agshubhs) [inline README.md:68]: @Divesh2
Add a row that shows the total in the table.

Comment (Divesh2) [reply README.md:68]: Sir, how can I clear commit msgs..?

File: README.md [hunk 5]
```diff
@@ -1,2 +1,79 @@
...
+## Mechanical Aspect of the Design
+
+
+Our bot has differential chasis with  three wheels out of which one is castor wheel and the other two motors are connected with two motors which provide movement to the bot.
+With the help of these wheels, the bot can move in all directions.The hands were operated using three dc motors.The "up-down" motion of hands were provided  using worm and spur gear mechanism.The hands were able to converge or diverge with the help of pulley and belt mechanism.
+The body was made using channels and plywood whereas the head was 3D printed.
+
+
+## Electronics Aspect of the Design
+We used two microcontrollers namely Arduino Uno and Arduino Nano, two L298n motor drivers, cytron motor driver,dc motors, servo motor,  bluetooth module and two LED Matrix MAX7219 for the this bot.Uno was used for movements of wheels and hands whereas nano was used for operating the head including LED Matrix used as the eyes of our bot, WallE.
```
Comment (agshubhs) [inline README.md]: Write the following section in points as a numbered list
1. Mechanical Aspect
2. Electronic aspect

File: README.md (hunk 4 above)
Comment (agshubhs) [reply README.md:68]: Now its not possible to change the commit messsage. In future take care of that

**Comment by agshubhs** [comment]
@Divesh2 Good Work. I am merging it and closing the PR.

**Pull request merged**